.github/workflows/ci.yml

```diff
@@ -0,0 +1,67 @@
+name: Continuous Integration
+
+on: [push, pull_request]
+
+jobs:
+  format:
+    name: Execute rustfmt
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Setup Rust toolchain
+        uses: actions-rs/toolchain@v1
+        with:
+          toolchain: nightly
+      - name: Execute rustfmt
+        run: cargo +nightly fmt
+
+  build:
+    name: Build the library
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Setup Rust toolchain
+        uses: actions-rs/toolchain@v1
+        with:
+          toolchain: stable
+      - name: Build the project
+        run: cargo build
+
+  test:
+    name: Execute the tests
```

Comment:
Will need a few runs to get this job right, I followed the same steps I did locally but hard to see if it will work in the GA server first time 😬